frontend/javascript/index.js

```diff
@@ -93,8 +93,20 @@ document.addEventListener("DOMContentLoaded", function(event) {
   });
 });
 
+function contextSave() {
+  var scrollPosition = localStorage.getItem('scrollPosition');
```

Comment:
this should be stored in `sessionStorage` instead of `localStorage` so that it isn't preserved across multiple site visits.